modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/EditObjectField.tsx

```diff
@@ -144,8 +139,7 @@ export default function EditObjectField({
 			readOnly={readOnly}
 			title={Liferay.Language.get('field')}
 		>
-			{(Liferay.FeatureFlags['LPS-170122'] && isDefaultStorageType) ||
-			values.businessType === 'Picklist' ? (
+			{isDefaultStorageType ? (
```

Comment:
The way that this part works in the master, a picklist have the 'advanced' tab, **even if** the storage type is nonDefault.
The way that you perform here, a picklist have no the 'advanced', if the storage type is nonDefault. 
It is your intention change this behavior?